src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -67,6 +61,62 @@ export default {
 		},
 		handleQuickFiltersOverlay() {
 		},
+		async fetchFilterData(loanSearchState = {}) {
+			// TODO: Prevent this from running on every query (not needed for sorting and paging)
+			const { isoCodes } = await runFacetsQueries(this.apollo, loanSearchState, FLSS_ORIGIN_CATEGORY);
+			const categories = await fetchCategories(this.apollo);
+
+			// Merge all facet options with filtered options
+			const facets = {
+				regions: transformIsoCodes(isoCodes, this.allFacets?.countryFacets),
+			};
+
+			this.quickFiltersOptions.categories = [
+				...[{ title: 'All categories', key: 0 }],
+				...categories.lend?.loanChannels?.values
+			];
+
+			this.quickFiltersOptions.location = facets.regions;
+
+			this.quickFiltersOptions.sorting = [
+				{
+					title: 'Recommended',
```

Comment:
I see we already have this sort array in `LoanChannelCategoryControl`. If these 2 components are supposed to always have the same sort options, it would be worth for maintenance putting that array in a util and sharing between these two components.

Reply:
We will have some tech debt work to refactor quick filters so it's more seamless to share - also this should pull from the API eventually

Reply:
I recommend adding a TODO here then for returning to pull this data from the API.